cmd/podman/farm/build.go

```diff
@@ -50,14 +49,8 @@ func init() {
 	cleanupFlag := "cleanup"
 	flags.BoolVar(&buildOpts.buildOptions.Cleanup, cleanupFlag, false, "Remove built images from farm nodes on success")
 
-	podmanConfig := registry.PodmanConfig()
 	farmFlagName := "farm"
-	// If remote, don't read the client's containers.conf file
-	defaultFarm := ""
-	if !registry.IsRemote() {
-		defaultFarm = podmanConfig.ContainersConfDefaultsRO.Farms.Default
-	}
-	flags.StringVar(&buildOpts.farm, farmFlagName, defaultFarm, "Farm to use for builds")
+	flags.StringVar(&buildOpts.farm, farmFlagName, "", "Farm to use for builds")
```

Comment:
How are we reading what the default farm is here now? Looks like all the logic around figuring out the default farm for build has been removed

Reply:
nvm, I see it below in pk/farm/farm.go